translations/hu-HU.json

```diff
@@ -0,0 +1,2970 @@
+{
+  "config": {
+    "name": "konfiguráció",
+    "description": "Kezeld a szervered konfigurációját."
+  },
+  "config view": {
+    "name": "megtekintés",
+    "description": "Tekintsd meg a szervered konfigurációját.",
+    "options": [],
+    "responses": {
+      "title.1": [
+        "{ctx.guild.name} konfigurációja"
+      ],
+      "description.1": [
+        "Automatikus válasz bekapcsolása: {autoreply}\n",
+        "Hypixel hírek ping: {news_ping}\n",
+        "Hypixel hírek csatornája: {news_channel}\n",
+        "SkyBlock frissítési jegyzetek: {patchnotes_ping}\n",
+        "SkyBlock frissítési jegyzetek csatornája: {patchnotes_channel}\n",
+        "Fetchur értesítések ping: {fetchur_ping}\n",
+        "Fetchur értesítések csatornája: {fetchur_channel}\n",
+        "Globális esemény ping: {global_event_ping}\n",
+        "Globális esemény csatornája: {global_event_channel}\n",
+        "SkyBlock dátum/idő csatornája: {skyblock_time}\n",
+        "SkyBlock események: {skyblock_events}\n",
+        "Műveleti napló: {scammer_logs}\n",
+        "Scammer művelet: {scammer_action}\n",
+        "IRL Kereskedő művelet: {trader_action}\n",
+        "Makrózó művelet: {macroer_action}\n",
+        "Rattelő művelet: {phisher_action}\n",
+        "\n",
+        "Módosítani tudod a konfigurációkat a '/config edit' paranccsal!"
```

Comment:
the command name here should be translated and placed in an inline codeblock (\`\`)